templates/common/infra/bicep/core/host/container-app.bicep

```diff
@@ -5,13 +5,19 @@ param tags object = {}
 param containerAppsEnvironmentName string
 param containerName string = 'main'
 param containerRegistryName string
+param secrets array = []
```

Comment:
@jongio I notice that we have a different approach in [keyvault-secret.bicep](https://github.com/Azure/azure-dev/blob/main/templates/common/infra/bicep/core/security/keyvault-secret.bicep). I don't know enough here about `bicep` handling without playing with it myself. Is the secret handling here working as expected?